releasenotes/notes/holland-vevn-9eb62140a2c8ff56.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - Holland now runs inside a python virtual environment
+    by default. By default, the virtual environment will
+    be placed in /openstack/venvs/holland-release-tag.
+upgrade:
+  - On upgrades, the holland pip packages will linger in 
+    the global environment if not cleaned up.
```

Comment:
This is what happens now. This is what happens even with OSA when upgrading from a version without virtual environments, to one with them. OSA does no clean-up and so this patch doesn't either.

Since 13.1 isn't being worked on *yet*, I would view the clean-up of *all* the kruft to be an upgrade task targeted to 13.1